packages/remix-server-runtime/routeModules.ts

```diff
@@ -63,8 +63,8 @@ export interface LinksFunction {
 /**
  * A function that loads data for a route.
  */
-export interface LoaderFunction {
-  (args: DataFunctionArgs):
+export interface LoaderFunction<Context = any> {
+  (args: DataFunctionArgs<Context>):
```

Comment:
`DataFunctionArgs` isn't currently a generic. Did you mean to update this? We'd also need to update `LoaderArgs` and `ActionArgs`.